README.md

```diff
@@ -1,8 +1,47 @@
-# Feedbacker-Forum
+<div align="center" ><img src="docs/images/feedbacker-logo.png" height="50%" width="50%"></div>
+
+---
+
+[![Build Status](https://travis-ci.org/ktk-2005/Feedbacker-Forum.svg?branch=develop)](https://travis-ci.org/ktk-2005/Feedbacker-Forum) [![License: MIT](https://img.shields.io/badge/License-MIT-yellow.svg)](LICENSE)
+
+[Feedbacker-Forum](https://feedbacker.site/about/) is a modern, open-source automated deployment pipeline solution for developers to help gather feedback easily. In other words, the developer can trigger a deployment process that ships a version of the current code for the product and gets a simple “feedback-able“ link in return.
+
+This link can be shared to peers that the developer craves feedback from.
+The developer can then create surveys, receive answers and discuss the live UI with peers.
+Feedbacker-Forum also supports creating "feedback-able" links for existing external websites.
+
+![Feedbacker example site](docs/images/user-survey-other.png)
+
+# Table of contents
+* [Setting Up](#Setting-Up)
+  * [Prerequisites](#Prerequisites)
+  * [Alternative docker setup](#Alternatively-setting-up-docker-container-for-dev-environment)
+* [Usage](#Usage)
+  * [Dashboard](#Dashboard)
+  * [Feedbacker](#Feedbacker)
+  * [Features](#Features)
+* [Contributing](#Contributing)
+  * [Running tests](#Running-tests)
+* [Credits](#Credits)
+* [License](#License)
+* [Screenshots](#Screenshots)
+  * [Demo](#demo)
+* [Future development roadmap](#Roadmap)  
+
 
 ## Setting up
 
-The project is built using [Node.js][node], currently actively tested with version 10.13.
+The application consist of a frontend, backend, proxy and database. The different parts can either be setup manually, or run together using containers.
+
+### Prerequisites
+
+The application is built using [Node.js][node], currently actively tested with version 10.13.
+The application also requires [go][golang] version 1.11 for running the proxy.
+Instances are run in containers with [docker][docker].
+
+If the application is run with containers [docker-compose][docker-compose] is also required for orchestrating the containers.
```

Comment:
What does this mean? Docker-compose is not required when not running the app backend itself outside of docker